src/20_Components/amp-font.html

```diff
@@ -0,0 +1,65 @@
+<!-- 
+  #### Introduction
+
+  `amp-font` allows to control timeouts on font loading on AMP HTML pages.
```

Comment:
We should explain what amp-font is good for, maybe: "Learn how to use the `amp-font` component to trigger and monitor the loading of custom fonts on AMP pages."